docs/Documentation/MkDocs.md

```diff
@@ -0,0 +1,20 @@
+# Write Documentation
+
+This documentation of JackTrip is generated with [Material](https://squidfunk.github.io/mkdocs-material/)
+for [MkDocs](https://www.mkdocs.org/). All pages are derived from Markdown files in the subdirectory `docs`
+in JackTrip's git repository. Setup and table of contents is found in a YAML file called mkdocs.yml in
+the root directory.
+
+If you only want to edit a page you can click on the pen symbol at the top of each page.
```

Comment:
Does that apply to locally served files or online?

Reply:
That's a feature that you can use when you are using MkDocs on services like GitHub, GitLab and maybe others. You can edit your pages directly on the Github website.